jep/1/README.adoc

```diff
@@ -0,0 +1,593 @@
+finished JEPs you may want to submit corrections to the JEP repository
+footnoteref:[repo] or the Jenkins issue tracker
+footnoteref:[issues,https://issues.jenkins-ci.org].  If the JEP author is a
+Jenkins developer, assign the bug/patch to them, otherwise assign it to a JEP
+editor.
+
+When in doubt about where to send your changes, please check first
+with the JEP author and/or a JEP editor.
+
+JEP authors with git push privileges for the JEP repository can update the
+JEPs themselves by using "git push" to submit their changes.
+
+
+==== Transferring JEP Ownership
+
+It occasionally becomes necessary to transfer ownership of JEPs to a
+new champion. In general, it is preferable to retain the original author as
+a co-author of the transferred JEP, but that's really up to the
+original author. A good reason to transfer ownership is because the
+original author no longer has the time or interest in updating it or
+following through with the JEP process, or has fallen off the face of
+the 'net (i.e. is unreachable or not responding to email). A bad
+reason to transfer ownership is because the author doesn't agree with the
+direction of the JEP. One aim of the JEP process is to try to build
+consensus around a JEP, but if that's not possible, an author can always
+submit a competing JEP.
+
+If you are interested in assuming ownership of a JEP, you can also do this via
+pull request. Fork the JEP repository, footnoteref:[repo] make your ownership
+modification, and submit a pull request. You should also send a message asking
+to take over, addressed to both the original author and the JEP editors via
+jenkinsci-dev@.  If the original author doesn't respond to email in a timely
+manner, the JEP editors will make a unilateral decision (it's not like such
+decisions can't be reversed :).
+
+
+==== JEP Editor Responsibilities & Workflow
+
+A JEP editor must subscribe to the jenkinsci-dev@googlegroups.com list and must
+watch the JEP repository footnoteref:[repo]. Most correspondence regarding JEP
+administration can be handled through GitHub issues and pull requests.
+
+For each new JEP that comes in an editor does the following:
+
+* Read the JEP to check if it is ready, sound, and complete. The ideas
+ must make technical sense, even if they don't seem likely to be
+ accepted.
+
+* The title should accurately describe the content.
+
+* Edit the JEP for language (spelling, grammar, sentence structure,
+ etc.), markup, code style.
+
+If the JEP isn't ready, an editor will send it back to the author for
+revision, with specific instructions.
+
+Once the JEP is ready for the repository, a JEP editor will:
+
+. Assign a JEP number (almost always just the next available number, but
+  sometimes it's a special/joke number, like 666 or 3141).
+. Create a new branch for the JEP, i.e. `jep-1`.
+. Retarget the original author's pull request to the new branch
+. Squash the commit(s) into the branch.
+. Update the JEP number in the document.
```

Comment:
OK I think that clarifies my question above. So we'd have as many branches as there JEPs, forever, right?